static/vendor/node-snackbar/index.d.ts

```diff
@@ -0,0 +1,42 @@
+declare module 'node-snackbar' {

```

Comment:
We don't need to save and maintain the TypeScript versions of the software, we can just use the compiled JavaScript.